torch/_inductor/codegen/triton.py

```diff
@@ -1712,7 +1746,36 @@ def store_reduction(self, name, index, value):
             DeferredLine(name, f"tl.store({var} + ({index}), {value}, {mask})")
         )
 
-    def scan(self, dtype, scan_op, value):
+    def _lift_helper(self, fn, num_args) -> str:
+        # Lift IR function into a triton function in the global namespace
+        name = f"_triton_helper_fn{len(self.helper_functions)}"
+        helper = IndentedBuffer()
+        helper.writeline("@triton.jit")
+        args = [f"arg{n}" for n in range(num_args)]
+        signature = ", ".join(args)
+        helper.writeline(f"def {name}({signature}):")
+
+        cse = CSE(prefix="", suffix="")
+        overrides = TritonOverrides(V.MockHandler())
+
+        class CSEProxy:
```

Comment:
This logic could probably be factored out and lifted to `common.py` to avoid duplicating it, but it's fine for now.